<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>引脚电平</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rad_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>pul</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>pos</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>431</width>
      <height>211</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>429</width>
       <height>209</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lab_pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
